client/lcd/lcd_test.go

```diff
@@ -629,7 +629,7 @@ func TestSubmitProposal(t *testing.T) {
 	require.Equal(t, uint32(0), resultTx.DeliverTx.Code)
 
 	var proposalID int64
-	cdc.UnmarshalBinaryBare(resultTx.DeliverTx.GetData(), &proposalID)
+	cdc.MustUnmarshalBinary(resultTx.DeliverTx.GetData(), &proposalID)
```

Comment:
These will now need merge conflict resolution off of the lateste `develop` due to the new amino changes.